src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/LicenseScanTests.cs

```diff
@@ -0,0 +1,243 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.SourceBuild.SmokeTests;
+
+[Trait("Category", "LicenseScan")]
+public class LicenseScanTests : TestBase
+{
+    private static readonly string[] s_allowedLicenseExpressions = new string[]
+    {
+        "apache-1.1", // https://opensource.org/license/apache-1-1/
+        "apache-2.0", // https://opensource.org/license/apache-2-0/
+        "apache-2.0 WITH llvm-exception", // https://foundation.llvm.org/relicensing/LICENSE.txt
+        "apsl-2.0", // https://opensource.org/license/apsl-2-0-php/
+        "bsd-new", // https://opensource.org/license/BSD-3-clause/
+        "bsd-original", // https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/bsd-original.LICENSE
+        "bsd-original-uc", // https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/bsd-original-uc.LICENSE
+        "bsd-simplified", // https://opensource.org/license/bsd-2-clause/
+        "bytemark", // https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/bytemark.LICENSE
+        "bzip2-libbzip-2010", // https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/bzip2-libbzip-2010.LICENSE
+        "cc0-1.0", // https://creativecommons.org/publicdomain/zero/1.0/legalcode
+        "cc-by-sa-3.0", // https://creativecommons.org/licenses/by-sa/3.0/legalcode
+        "cc-by-sa-4.0", // https://creativecommons.org/licenses/by-sa/4.0/legalcode
+        "cc-pd", // https://creativecommons.org/publicdomain/mark/1.0/
+        "cecill-c", // https://cecill.info/licences/Licence_CeCILL-C_V1-en.txt
```

Comment:
This seems to be a false positive?

Reply:
Fixed in https://github.com/dotnet/installer/pull/17442/commits/80e9a0360755f86a1b8fc97d2a136da161b2fd36